README.md

```diff
@@ -1,33 +1,40 @@
-# **idb-pconnector - Promise based DB2 Connector for IBM i**
+# **idb-pconnector - Promise based DB2 Connector for IBM i** <!-- omit in toc -->
 
-**Project Status**: (production-ready as a "technology preview")
+**Project Status**: (production ready as a "technology preview")
 
-The objective of this project is to provide a promise based database connector API for DB2 on IBM i.
+**Objective**: provide a promise based database connector for DB2 on IBM i.
 
 
-This project is a wrapper over the [`idb-connector`](https://bitbucket.org/litmis/nodejs-idb-connector) project but returning promises instead of using callbacks.
+This project is a promise based wrapper over the [`idb-connector`](https://bitbucket.org/litmis/nodejs-idb-connector) project.
```

Comment:
Need to adjust URL to point to IBM GitHub

Reply:
Good catch thanks!

Reply:
Just some minor grammatical changes throughout this file, like 'promise based' should be 'promise-based.' (Lines 1, 5, 8). Also, should we maybe mention that it enables async/await while also tooting the modern chops of the packages? Something like...

This project is a promised-based wrapper over the [`idb-connector`](https://github.com/IBM/nodejs-idb-pconnector) project that enables the use of modern JavaScript's [async/await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) syntax.

Also, why is there a blank line between 13 and 15?

Reply:
updated with requested changes